scalardb-analytics-postgresql-sample/README.md

```diff
@@ -0,0 +1,233 @@
+# A Sample Application of ScalarDB Analytics with PostgreSQL
+
+This pages describe [the sample application](https://github.com/scalar-labs/scalardb-samples/scalardb-analytics-postgresql) of ScalarDB Analytics with PostgreSQL.
+
+## Setting up the database
+
+First, you need to set up the database to run analytical queries with ScalarDB Analytics with PostgreSQL. If you haven't done it yet, please follow the instructions in [the Getting Started page](./getting-started.md).
```

Comment:
@josh-wong I've just noticed the same link issue here 😓. This link should point to the Getting Started page of  ScalarDB Analytics, which will be created with [this PR](https://github.com/scalar-labs/scalardb-analytics-postgresql/pull/14). Since the URL of the page doesn't exist yet, do you know how I should add the link to the page?

Reply:
@choplin The following URL should work once the "Getting Started" doc is live!👍

https://scalardb.scalar-labs.com/docs/3.9/scalardb-analytics-postgresql/getting-started.md